fala/apps/adviser/tests/test_search_view_function.py

```diff
@@ -70,6 +74,13 @@ def test_category_search_heading(self):
             html=True,
         )
 
+    def test_search_parameters_box_contains_only_postcode_and_categories(self):
+        response = self.client.get(self.url, self.data)
+        soup = bs4.BeautifulSoup(response.content)
+        search_params_box = soup.find("div", class_="laa-fala__grey-box")
+        content = search_params_box.text.replace("\n", "")
```

Comment:
Can you just add a comment to this text to say why you're removing spaces, hence why the string check below is slightly clumped together in the assertEquals. I know why, because of how the HTML is displaying the Strings, but someone else in the future may get confused.